tests/Unit/Service/AmazonWebServices/S3AdapterFactoryTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * CRM PROJECT
+ * THIS FILE IS A PART OF CRM PROJECT
+ * CRM PROJECT IS PROPERTY OF Legal One GmbH.
+ *
+ * @copyright Copyright (c) 2020 Legal One GmbH (http://www.legal.one)
```

Comment:
was this file copied or maybe it is unnecessary?

Reply:
HAHA, this is my IDE configuration, disabled the configuration for this project.

Reply:
ok, thanks for fix, rebased with master, so if everything will be working we will merge this :wink: